src/main/java/edu/cornell/kfs/module/purap/batch/JaggaerXMLInputFileType.java

```diff
@@ -0,0 +1,76 @@
+package edu.cornell.kfs.module.purap.batch;
+
+import edu.cornell.kfs.module.purap.CUPurapConstants;
+import edu.cornell.kfs.module.purap.CUPurapKeyConstants;
+import edu.cornell.kfs.module.purap.jaggaer.supplier.xml.SupplierSyncMessage;
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.core.api.datetime.DateTimeService;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.batch.XmlBatchInputFileTypeBase;
+
+import java.io.File;
+
+import static org.kuali.kfs.pdp.PdpConstants.FILE_NAME_PART_DELIMITER;
+
+public class JaggaerXMLInputFileType extends XmlBatchInputFileTypeBase<SupplierSyncMessage> {
```

Comment:
Should we add additional unit testing so we can validate this is doing what we expect?

Reply:
I've added a few unit tests